game/src/common/waypoints.rs

```diff
@@ -69,22 +69,22 @@ impl InputWaypoints {
         let mut delete_buttons = Vec::new();
 
         for (idx, waypt) in self.waypoints.iter().enumerate() {
-            let batch = Text::from(Line(format!("{}) {}", waypt.order, waypt.label))).render(ctx);
-            let bounds = batch.get_bounds();
-            let image = Image::from_batch(batch, bounds)
-                .color(RewriteColor::NoOp)
+            let waypoint = ctx
```

Comment:
This is _much_ cleaner, thank you for fixing. Using button builders just to grab the batch makes way more sense than trying to replicate that style with lower-level APIs like Text.